docs/customization.rst

```diff
@@ -1,11 +1,36 @@
 =====================
-Customizing ``xonsh``
+Updating and customizing xonsh
 =====================
```

Comment:
The `===` need to go all the way below and above the title text.

Reply:
Sure, makes sense. Will push the update in  a minute.